src/routes/hot.svelte

```diff
@@ -18,14 +18,19 @@
 </svelte:head>
 
 <main>
-	<header>
-		<img src="{base}/fire.png" alt="Fire logo" />
-		<h1>List of the hottest<br /> vacancies so far.</h1>
-	</header>
-	<div class="vacancies-list" id="vacancies-list">
-		{#each vacancies as vacancy}
-			<Vacancy data={vacancy} />
-		{/each}
+	<div class="app-container">
+		<div id="logo-container">
+			<a href="{base}/"><img src="{base}/logo-black.png" alt="JD logo" /></a>
+		</div>
+		<header>
+			<img src="{base}/fire.png" alt="Fire logo" />
+			<h1>List of the hottest<br /> vacancies so far.</h1>
+		</header>
+		<div class="vacancies-list" id="vacancies-list">
+			{#each vacancies as vacancy}
+				<Vacancy data={vacancy} />
+			{/each}
+		</div>
 	</div>
```

Comment:
`.app-container` selector is useless, as whole page is already wrapped in `<main>`

Reply:
If I remove `.app-container`, page `Hot vacancies` will have strange view.